TERMINOLOGY.md

```diff
@@ -1,53 +1,70 @@
-# Terminology
+# Concepts and terminology
 
 ## Estimators and predictors
+
 Fairlearn largely follows the [terminology established by scikit-learn](https://scikit-learn.org/stable/developers/contributing.html#different-objects), specifically:
-- Estimator: implements a `fit` method
-- Predictor: implements a `predict` method.
-Unlike [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator) our estimators in fairlearn may not be deterministic. Lots of disparity mitigation techniques require randomized output to meet their objective in expectation. We do provide unrandomized versions of every method as well, but they may not be able to uphold the same theoretical guarantees.
+- _Estimators_ implement a `fit` method.
+- _Predictors_ implement a `predict` method.
+
+
+**Randomization.** Unlike [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator), estimators in fairlearn can produce randomized predictors. Randomization of predictions is required to satisfy many definitions of fairness. Because of randomization, it is possible to get different outputs from the predictor's `predict` method on identical data. For each of our methods, we provide explicit access to the probability distribution used for randomization.
+
+## Fairness of AI systems
+
+AI systems can behave unfairly for a variety of reasons. Sometimes it is because of societal biases reflected in the training data and in the decisions made during the development and deployment of these systems. In other cases, AI systems behave unfairly not because of societal biases, but because of characteristics of the data (e.g., too few data points about some group of people) or characteristics of the systems themselves. Because it can be hard to distinguish between these reasons (and these reasons are not mutually exclusive and often exacerbate one another), we define whether an AI system is behaving unfairly in terms of its impact on people – i.e., in terms of harms – and not in terms of specific causes, such as societal biases, or in terms of intent, such as prejudice.
+
+**Usage of the word _bias_.** Since we define fairness in terms of harms rather than specific causes (such as societal biases), we avoid the usage of the words _bias_ or _debiasing_ in describing the functionality of fairlearn.
+
+### Types of harms
+
+There are many types of harms (see, e.g., the [keynote by K. Crawford at NIPS 2017](https://www.youtube.com/watch?v=fMym_BKWQzk)). Fairlearn is most applicable to two kinds of harms:
```

Comment:
We need to standardize the way we write fairlearn:
- fairlearn
- Fairlearn
- FairLearn
are all possible, but it should be uniform. I'm actually for `fairlearn` since that's consistent with the code, but open to anything.

@bethz @riedgar-ms FYI

Reply:
I'd vote for `fairlearn` too

Reply:
I'm in favor of treating it as a noun, but not a proper noun, so "fairlearn" in the middle of the sentence, but "Fairlearn" at the beginning of the sentence. Inside URLs and file paths, I think we should write "fairlearn". Thoughts?

Reply:
But `fairlearn` is a proper noun - our "brand" (for want of a better word). So even if starting a sentence, it should not be capitalised.

We could cheat, and make sure we never begin sentences with it...

Reply:
resolving to all lowercase fairlearn